pages/investigate.py

```diff
@@ -7,6 +7,7 @@
 from clarifai.client import create_stub
```

Comment:
Please make sure to change the imports to new SDK
```python
from clarifai.client.auth import create_stub
from clarifai.client.auth.helper import ClarifaiAuthHelper
```

Reply:
Modified.

Reply:
This is not yet modified

`from clarifai.client.auth import create_stub`

Reply:
I meant both as commented on first
for any auth related imports it should be from
`clarifai.client.auth` as we will remove `client.auth` in near future

Reply:
Hi @sainivedh, 
Some problem with my vscode, that's why it was not committed before. Now I have modified it. Kindly check now.